docs/install_libs.md

```diff
@@ -0,0 +1,10 @@
+# Optimized libraries
+
+Since Host OS focus is on virtualization, you might need some
+additional packages that are not provided by this project, for
+example, optimized libraries to take advantage of hardware functions
+or a more recent toolchain.
+
+In this case, we suggest that you install the Advance Toolchain for
+Linux on Power (AT). The instructions to install AT are described
```

Comment:
I suggest: To install AT on top of Host OS or CentOS 7, follow the instructions for RHEL 7 [here].